cartographer_ros/cartographer_ros/submap.cc

```diff
@@ -30,29 +30,29 @@ std::unique_ptr<SubmapTexture> FetchSubmapTexture(
   ::cartographer_ros_msgs::SubmapQuery srv;
   srv.request.trajectory_id = submap_id.trajectory_id;
   srv.request.submap_index = submap_id.submap_index;
-  if (!client->call(srv)) {
+  if (!client->call(srv) || srv.response.textures.empty()) {
```

Comment:
We don't want to send empty texture lists, so this would be a bug. Similar to before: Make this a `CHECK` instead of failing silently?

Reply:
Done